lua/toggleterm/terminal.lua

```diff
@@ -287,10 +290,11 @@ end
 
 ---Combine arguments into strings separated by new lines
 ---@vararg string
+---@param newline_chr string
 ---@return string
-local function with_cr(...)
+local function with_cr(newline_chr, ...)
   local result = {}
-  local newline_chr = get_newline_chr()
+  -- local newline_chr = get_newline_chr()
```

Comment:
should this be commented out? or removed?

Reply:
I think this can safely be removed.